etl/steps/data/garden/un/2024-01-17/urban_agglomerations_300k.meta.yml

```diff
@@ -0,0 +1,126 @@
+  share_urban_from_tot: &share_urban_from_tot Measures the percentage of a country's total population living in urban agglomerations with 300,000 inhabitants or more.
+  urban_pop: &urban_pop Refers to the annual population figures for urban agglomerations with 300,000 inhabitants or more.
+
+  projections_key: &projections_key
+    - The projection method for urban population in the World Urbanization Prospects involves a two-step process using an established extrapolation method based on urban-rural ratios.
+    - Initially, the average annual rate of change in the urban-rural ratio is calculated using data from the last two censuses, which informs the rate of change in urban and rural populations.
+    - This rate is then extrapolated, assuming a logistic path of urban proportion growth.
+    - Subsequently, a "world norm" is applied, estimated from empirical urban-rural growth differences in two groups of countries categorized by population size.
+    - This norm uses a regression equation to establish a hypothetical urban-rural growth difference for different levels of initial urban percentage.
+    - The country-specific urban-rural growth difference is then converged with this hypothetical difference over 25 years, allowing the urbanization process of a country to align with a global urbanization pattern.
+    - This method ensures that urban-rural growth differences evolve towards a worldwide trend rather than remaining constant.
+  descr_producer: &descr_producer The term “urban agglomeration” refers to the population contained within the contours of a contiguous territory inhabited at urban density levels without regard to administrative boundaries. It usually incorporates the population in a city or town plus that in the suburban areas lying outside of, but being adjacent to, the city boundaries. Whenever possible, data classified according to the concept of urban agglomeration are used. However, some countries do not produce data according to the concept of urban agglomeration but use instead that of metropolitan area or city proper. If possible, such data are adjusted to conform to the concept urban agglomeration. When sufficient information is not available to permit such an adjustment, data based on the concept of city proper or metropolitan area are used.
+
+
+dataset:
+  title: World Urbanization Prospects Dataset - Urban Agglomerations with 300,000 or More Inhabitants
+
+tables:
+  urban_agglomerations_300k:
+    variables:
+      #
+      #  Estimates
+      #
+      average_annual_rate_of_change_of_urban_agglomerations_with_300_000_inhabitants_or_more_in_2018__by_country__percent_estimates:
+        title: Average annual rate of change cities with 300,000 inhabitants or more (estimates)
+        unit: "%"
+        short_unit: "%"
+        description_short: *rate_of_change
+        description_from_producer: *descr_producer
```

Comment:
Nit - since `description_from_producer` is identical for all indicators, it could go to `common` section.